File: tests/openmp_examples/devices/target_associate_ptr.1.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp target update to(arr[ioff:50]) device(dev)
#pragma omp target map(tofrom : arr[ioff:50]) device(dev)
#pragma omp target update from(arr[ioff:50]) device(dev)

File: tests/openmp_examples/devices/target_associate_ptr.1.f90
!$omp     target update to(arr(ioff:ioff+CS-1)) device(dev)
!$omp     target map(tofrom: arr(ioff:ioff+CS-1)) device(dev)
!$omp     end target
!$omp     target update from(arr(ioff:ioff+CS-1)) device(dev)
